src/main/java/racingcar/domain/car/CarName.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.domain.car;
+
+import static racingcar.domain.exception.CarExceptionMessage.BLANK;
+import static racingcar.domain.exception.CarExceptionMessage.CONTAIN_WRONG_ELEMENT;
+import static racingcar.domain.exception.CarExceptionMessage.OUT_OF_LENGTH;
+
+import java.util.Objects;
+
+public class CarName {
+    private static final String WRONG_ELEMENT = ",";
+    private static final int MAX_LENGTH = 5;
+    private final String name;
+
+    public CarName(String name) {
+        validate(name);
+        this.name = name;
+    }
+
+    private void validate(String name) {
+        validateBlank(name);
+        validateWrongExists(name);
+        validateLength(name);
+    }
+
+    private void validateBlank(String name) {
+        if (name == null || name.isBlank()) {
+            throw new IllegalArgumentException(BLANK.getMessage());
+        }
+    }
+
+    private void validateWrongExists(String name) {
+        if (name.contains(WRONG_ELEMENT)) {
+            throw new IllegalArgumentException(CONTAIN_WRONG_ELEMENT.getMessage());
+        }
+    }
+
+    private void validateLength(String name) {
+        if (name.isEmpty() || name.length() > MAX_LENGTH) {
```

Comment:
`name.isEmpty()` 부분은 이미 위의 `validateBlank` 부분의 `name.isBlank()`에서 검증이 되는 것으로 알고 있습니다!
중복 로직을 조금 제거할 수 있을 것 같아요!
```suggestion
        if (name.length() > MAX_LENGTH) {
```